suitcase/mongo_normalized/tests/tests.py

```diff
@@ -12,3 +12,43 @@ def test_export(db_factory, example_data):
     serializer = Serializer(metadatastore_db, asset_registry_db)
     for item in documents:
         serializer(*item)
+
+
+def listit(t):
+    return list(map(listit, t)) if isinstance(t, (list, tuple)) else t
+
+
+def test_update(db_factory, example_data):
+    documents = example_data()
+    metadatastore_db = db_factory()
+    asset_registry_db = db_factory()
+    serializer = Serializer(metadatastore_db, asset_registry_db)
+    for item in documents:
+        serializer(*item)
+    origin = documents[0][1]
+    if 'hints' in origin.keys():
```

Comment:
This is clever, but fragile. If tuples are added it other places in the document in the future it will break. We normally use the approach

```py
from event_model sanitize_doc
...
assert sanitize_doc(real) == sanitize_doc(start)
```